packages/core/src/queries/user-sso-identities.ts

```diff
@@ -24,9 +25,19 @@ export default class UserSsoIdentityQueries extends SchemaQueries<
   ): Promise<Nullable<UserSsoIdentity>> {
     return this.pool.maybeOne<UserSsoIdentity>(sql`
       select *
-      from ${UserSsoIdentities.table}
-      where ${UserSsoIdentities.fields.issuer} = ${issuer}
-      and  ${UserSsoIdentities.fields.identityId} = ${ssoIdentityId}
+      from ${sql.identifier([UserSsoIdentities.table])}
+      where ${sql.identifier([UserSsoIdentities.fields.issuer])} = ${issuer}
+      and ${sql.identifier([UserSsoIdentities.fields.identityId])} = ${ssoIdentityId}
```

Comment:
Is it necessary to wrap `identifier`?

Reply:
yes, added the new query method and API integration tests. The syntax error was found by IT test cases and we wrapped the identifier to address this syntax error.